deployment/ansible/playbook.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes cache_valid_time=3600
+
+  roles:
+    - { role: "azavea.terraform" }
+    - { role: "azavea.java" } 
```

Comment:
Java is necessary for `sbt` commands. This was added to keep the development environment working until we got the container infrastructure functional.